.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
     - TOX_SUFFIX="httplib2"
     - TOX_SUFFIX="boto3"
```

Comment:
Below in the `allowed_failures` there is still `boto3` and travis is failing on this environment. So this doesn't address #382 from what I'm seeing.